plugins/package-managers/bitbake/README.md

```diff
@@ -0,0 +1,13 @@
+# About
+
+This is a package manager plugin for the [OSS Review Toolkit][ORT] to analyze [Yocto] projects managed by [BitBake].
+It supersedes the combination of the [meta-doubleopen] and [do-convert] projects by relying on upstream [SBOM] generation in [SPDX] format, and converting the generated files to an ORT analyzer result file via ORT's [SPDX document file analyzer].
+
+[ORT]: https://github.com/oss-review-toolkit/ort
+[BitBake]: https://docs.yoctoproject.org/bitbake.html
+[Yocto]: https://www.yoctoproject.org/
+[meta-doubleopen]: https://github.com/doubleopen-project/meta-doubleopen
+[do-convert]: https://github.com/doubleopen-project/do-convert
+[SBOM]: https://docs.yoctoproject.org/dev/dev-manual/sbom.html
+[SPDX]: https://spdx.dev/
+[SPDX document file analyzer]: https://oss-review-toolkit.org/ort/docs/tools/analyzer
```

Comment:
commit: Why did you choose to use the analyzer for mapping the SPDX file? 

(I don't know that analyzer inside out, but I recall that it was implemented with some specific use case in mind which is beyond the SDPX spec)

Reply:
> Why did you choose to use the analyzer for mapping the SPDX file?

That was the easiest way to make use of `SpdxDocumentCache` / `SpdxResolvedDocument` and related resolution logic right now.

> I recall that it was implemented with some specific use case in mind which is beyond the SDPX spec

Yes, indeed we might run into issues here when fully implementing document resolution. Ideally, any such issues could be fixed in `SpdxDocumentFile` itself (e.g. by generalizing code or making some behavior configurable). But if that does not work out, we should consider making direct use of the "low-level" `spdx-utils` in `BitBake` instead.